src/mlpack/methods/softmax_regression/softmax_regression_impl.hpp

```diff
@@ -14,47 +14,59 @@ namespace mlpack {
 namespace regression {
 
 template<template<typename> class OptimizerType>
+SoftmaxRegression<OptimizerType>::
+SoftmaxRegression(const size_t inputSize,
+                  const size_t numClasses,
+                  const bool fitIntercept) :
+  inputSize{inputSize},
+  numClasses{numClasses},
+  lambda{0.0001},
+  fitIntercept{fitIntercept}
```

Comment:
For consistency, do you mind if I change this to `()` initialization vs. `{}` initialization?